app/docker/rest/service.js

```diff
@@ -22,7 +22,7 @@ function ServiceFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider, Htt
       ignoreLoadingBar: true
     },
     update: {
-      method: 'POST', params: { id: '@id', action: 'update', version: '@version' },
+      method: 'POST', params: { id: '@id', action: 'update', version: '@version', rollback: '@rollback' },
       headers: {
         // TODO: This is a temporary work-around that allows us to leverage digest pinning on
```

Comment:
Well, as long as this work-around is documented in https://github.com/portainer/portainer/issues/1987 so that we do know what we need to change, I'm ok to remove it.

Reply:
https://github.com/portainer/portainer/issues/1987#issuecomment-518647649